authentication/views/sysadmin.py

```diff
@@ -73,11 +160,42 @@ def create_user(request):
                 + user.email
                 + ".",
             )
-            return HttpResponseRedirect(reverse("authentication:create-user"))
+            return HttpResponseRedirect(reverse("authentication:user-directory"))
```

Comment:
I think there is a bug here with the success message and the redirect (maybe I didn't implement the sucesss message here properly?). Try the following:
1. create a teacher
2. you are redirected to the user directory
3. go back to the create a teacher form
4. you can see the success message from creation of 1.